cypress/integration/models/businessservices.ts

```diff
@@ -0,0 +1,149 @@
+        clickByText(navTab, businessservices);
+    }
+
+    protected fillName(name: string): void {
+        inputText(businessServiceNameInput, name);
+    }
+
+    protected fillDescription(description: string): void {
+        inputText(businessServiceDescriptionInput, description);
+    }
+
+    protected selectOwner(owner: string): void {
+        selectFormItems(businessServiceOwnerSelect, owner);
+    }
+
+    getBusinessServiceName(): string {
+        this.businessServiceName = faker.company.companyName();
+        return this.businessServiceName;
+    }
+
+    getBusinessServiceDescription(): string {
+        this.businessServiceDescription = faker.lorem.sentence();
+        return this.businessServiceDescription;
+    }
+
+    create({ owner = null, cancel = false } = {}): void {
+        BusinessServices.clickStakeholdergroups();
+        clickByText(button, createNewButton);
+        if (cancel) {
+            cancelForm();
+        } else {
+            this.getBusinessServiceName();
+            this.getBusinessServiceDescription();
+            this.fillName(this.businessServiceName);
+            this.fillDescription(this.businessServiceDescription);
+            if (owner) {
+                this.selectOwner(owner);
+            }
+            submitForm();
+            checkSuccessAlert(
+                successAlertMessage,
+                `Success! ${this.businessServiceName} was added as a business service.`
+            );
+        }
+    }
+
+    edit({
+        name = this.businessServiceName,
+        description = this.businessServiceDescription,
+        owner = null,
+        cancel = false,
+    } = {}): void {
+        BusinessServices.clickStakeholdergroups();
+        selectItemsPerPage(100);
+        cy.wait(2000);
+        cy.get(tdTag)
+            .contains(this.businessServiceName)
+            .parent(trTag)
+            .within(() => {
+                click(editButton);
+            });
+
+        if (
+            !cancel &&
+            (name !== this.businessServiceName ||
+                description !== this.businessServiceDescription ||
+                owner)
+        ) {
+            this.fillName(name);
+            this.fillDescription(description);
+            if (owner) {
+                this.selectOwner(owner);
+            }
+            submitForm();
+            this.businessServiceName = name;
+        } else {
+            cancelForm();
+        }
+    }
+
+    delete({ name = this.businessServiceName, cancel = false } = {}): void {
+        BusinessServices.clickStakeholdergroups();
+        selectItemsPerPage(100);
+        cy.wait(2000);
+        cy.get(tdTag)
+            .contains(name)
+            .parent(trTag)
+            .within(() => {
+                click(deleteButton);
+            });
+        if (cancel) {
+            cancelForm();
+        } else {
+            click(confirmButton);
+        }
+    }
+
+    exists({ name = this.businessServiceName } = {}) {
```

Comment:
Much better, this argument should be compulsory or default, but not optional. We can do this for previously added methods in stakeholder and stakeholdergroups
+1